lib/src/core.rs

```diff
@@ -10,9 +11,11 @@ pub enum ProcessResult {
     Done(NodeData),
 }
 
-#[derive(Clone)]
+#[derive(Clone, Getters, Debug)]
 pub struct Core {
+    #[get = "pub"]
```

Comment:
```suggestion
    #[get_copy = "pub with_prefix"]
```

Reply:
I have never know about this before. Thank you very much.